tests/processes/test_wps_package.py

```diff
@@ -494,3 +500,155 @@ def test_merge_io_formats_wps_overlaps_cwl():
         Format(CONTENT_TYPE_APP_XML),
         Format(CONTENT_TYPE_TEXT_PLAIN),
     ])
+
+
+def test_stdout_stderr_logging_for_commandline_tool_success():
```

Comment:
add a docstring that says the evaluated criteria of the test so we can understand the purpose at a glance